src/dreyfus_index_manager.erl

```diff
@@ -119,7 +124,15 @@ handle_db_event(DbName, created, _St) ->
     gen_server:cast(?MODULE, {cleanup, DbName}),
     {ok, nil};
 handle_db_event(DbName, deleted, _St) ->
-    gen_server:cast(?MODULE, {cleanup, DbName}),
+    DoRecovery = config:get_boolean("couchdb",
```

Comment:
`RecoveryEnabled` would be a better name, we're not 'doing' recovery here, we're just enabling a future recovery.